### DISCUSSION TITLE
[Bug] Sparkle Slime Balloon doesn't work right (Added buff to NPC abnormally)

### DISCUSSION THREAD

**Discussion by grechnevi** [Pryaxis/TShock#1898] [Q&A] [answered]
[Sparkle Slime Balloon](https://terraria.gamepedia.com/Sparkle_Slime_Balloon) can't be thrown to NPC as it causes kick with followed text in console logs:
> TSPlayer: INFO: Kicked Username for : 'Added buff to NPC abnormally.'

**Comment by hakusaro** [comment, marked as answer]
This is fixed in the latest development builds. A pre-release will be out soon.
